lib/omnibus-ctl.rb

```diff
@@ -694,6 +694,11 @@ def run(args)
 
       command_to_run = args[0]
 
+      if command_to_run == "--help"
+        help
+        Kernel.exit 0
+      end
```

Comment:
If it can follow the same code path as `help` with that change ^^, that is clearer to me, thanks!